### DISCUSSION TITLE
Why bionic works so slow...

### DISCUSSION THREAD

**Discussion by sergeymeleschenko** [bionic-gpt/bionic-gpt#423] [General]
### Describe your issue

Why bionic works so slow... with 32CPU & 48 RAM ? whta can i do ?

### Describe your setup/startup scripts

docker-compose up -d docker-compose.yaml

Any tell me the way how can i use another model? for example llama2:70b?

### Steps to reproduce

type any questions to chat and wait a lot of time for the answer

**Comment by mohamedkassem23** [comment]
Hi
I think you are running a 70b model this requires a powerful GPU try to run light 7b models, not 70b
